src/ClientData/CallstackDataTest.cpp

```diff
@@ -200,4 +200,124 @@ TEST(CallstackData, FilterCallstackEventsBasedOnMajorityStartExcludesFunctionToS
                                                                          event4, event5, event6}));
 }
 
+TEST(CallstackData, ForEachCallstackEventOfTidInTimeRangeDiscretized) {
+  CallstackData callstack_data;
+
+  const uint64_t cs_id = 12;
+  CallstackInfo cs{{0x11, 0x10}, CallstackType::kComplete};
+  callstack_data.AddUniqueCallstack(cs_id, std::move(cs));
+
+  constexpr uint32_t kTid = 42;
+  constexpr uint32_t kAnotherTid = 43;
```

Comment:
I'd prefer to put these right above the `TEST...`, so they would get static storage duration. But TBH it doesn't matter here.